staging/src/k8s.io/dynamic-resource-allocation/resourceslice/resourceslicecontroller.go

```diff
@@ -73,12 +73,14 @@ type Controller struct {
 	resources *DriverResources
 }
 
+// +k8s:deepcopy-gen=true
```

Comment:
/lgtm cancel

Please insert an empty line between this special comment and the normal Go docs.

Reply:
done